src/Stepper/Step.tsx

```diff
@@ -0,0 +1,119 @@
+import * as classnames from "classnames";
+import * as React from "react";
+import * as PropTypes from "prop-types";
+
+import "./Step.less";
+import CheckMark from "./CheckMark";
+import {FlexBox} from "../flex";
+import Exclamation from "./Exclamation";
+
+const propTypes = {
+  className: PropTypes.string,
+  title: PropTypes.string,
+  description: PropTypes.node,
+  id: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
```

Comment:
1 - we can keep this simple and just make it a string (i wouldn't encourage numeric IDs anyway)
2 - make this prop required